src/app/person/person.hpp

```diff
@@ -23,4 +31,5 @@ class Person
     void setPesel(const std::string& pesel);
     std::string getPesel() const;
     bool validatePesel(const std::string& PESEL) const;
+    bool validatePesel() const;
```

Comment:
I suggest moving validation methods to private and use them properly within object.
`setPesel(pesel)
{
if(validatePesel(pesel))
{
 pesel_=pesel;
}
else {
throw exception;
}
}`

Reply:
Done